Makefile

```diff
@@ -51,3 +56,29 @@ test-functional:
 freeze:
 	$(call print_msg, Freezing... )
 	python setup.py build
+
+
+# Use platform flag to build multi-platform pex (requires platform wheels).
+# --platform=macosx-10.12-x86_64 --platform=linux_x86_64 --platform=win64_amd
+WHEEL_CACHE := $(PWD)/dist/wheels
+PEX_ARGS    := -v --no-pypi --cache-dir=$(WHEEL_CACHE)
+
+clean:
+	rm -rf $(WHEEL_CACHE)
+	rm -f dist/clusterrunner
+
+# TODO: Consider using "pip download --platform" when direct downloading
+#       os cross-platform wheels is supported.
+.PHONY: wheels
```

Comment:
Is it a common practice to split up these `.PHONY` directives throughout the Makefile? I didn't realize you could do that but even though it's a bit more verbose it seems nicer since it will probably get (correctly) copy/pasted when new make targets are added.

1. Should we do that with the other existing targets to keep a consistent style through the file?
2. Does "clean" need a phony?